### PR TITLE
[[Documentation]] transfer-mode - modes added

### PR DESCRIPTION
**Pull request by seaniepie**
The list of transfer modes added for easy reference

### CODE REVIEW

File: docs/glossary/t/transfer-mode.lcdoc [hunk 1]
```diff
@@ -12,6 +12,35 @@ Some transfer modes make the <object(glossary)> partly or completely
...
+* <srcBic(keyword)>
+* <srcCopy(keyword)>
+* <srcOr(keyword)>
+* <srcOrReverse(keyword)>
+* <srcXOr(keyword)>
+* <subOver(keyword)>
+* <subPin(keyword)>
+* <transparent(keyword)>
+
 References: object (glossary), pixel (glossary)
```
Comment (livecodeali) [inline docs/glossary/t/transfer-mode.lcdoc:44]: @seaniepie All of the transfer mode keywords need to be added to the `References:`. Also, in the bullet list you don't need to include the type `(keyword)` as the link specified in the references will be non-ambiguous! You only have to do that if there are two references with the same name of different type (as frequently occurs with `button (glossary)` and `button (object)` for example)

**Review by livecodeali — CHANGES_REQUESTED** [review]

File: docs/glossary/t/transfer-mode.lcdoc (hunk 1 above)
Comment (seaniepie) [reply docs/glossary/t/transfer-mode.lcdoc:44]: Done this, ready to go. Thanks for all your guidance. My head is finally getting around all of this. I must be getting old.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 72f5eab

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 72f5eab2bbe1cb38f97ea924c01a2b0f03b67f57

**Comment by livecode-vulcan** [comment]
😎 test success 72f5eab2bbe1cb38f97ea924c01a2b0f03b67f57
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/351)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/356)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/351)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/351)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/357)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/351)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/351)

**Review by livecodeali — APPROVED** [review]

**Pull request merged**